video/out/opengl/common.c

```diff
@@ -274,6 +274,23 @@ static const struct gl_functions gl_functions[] = {
         },
     },
     {
+        .ver_core = 320,
+        .extension = "GL_ARB_timer_query",
+        .functions = (const struct gl_function[]) {
+            DEF_FN(GenQueries),
+            DEF_FN(DeleteQueries),
+            DEF_FN(BeginQuery),
+            DEF_FN(EndQuery),
+            DEF_FN(QueryCounter),
+            DEF_FN(IsQuery),
+            DEF_FN(GetQueryObjectiv),
+            DEF_FN(GetQueryObjecti64v),
+            DEF_FN(GetQueryObjectuiv),
+            DEF_FN(GetQueryObjectui64v),
```

Comment:
> What is that?

This one: https://www.opengl.org/sdk/docs/man/html/glGetQueryObject.xhtml

> When in doubt, check the specs. They're authoritative.

Okay. Spec says it's not available in 3.2